plugins/LinuxVST/README.md

```diff
@@ -48,9 +60,15 @@ include/vstsdk/
 └── vstplugmain.cpp
 ```
 
-Then, navigate to the `build` directory and run `cmake .. && make`. A number of
-`.so` files should appear — these are your plugins. Copy them wherever you
-would like or wherever your DAW can find them.
+
+Then, build and install:
+```
+cd airwindows/plugins/LinuxVST/
+cmake
+make
+mkdir -p /usr/local/lib/lxvst
+cp *.so /usr/local/lib/lxvst
```

Comment:
Most Linux VST hosts actually look  for LinuxVSTs in `/usr/lib/vst` and many distributions install them there.

Reply:
Some distributions install the vst in /isr/lib64/vst ... Using cmake variable LIBDIR.